specification/amqp_0.9.1/src/main/scripts/org/kaazing/specification/amqp/ws/open/identity/response.rpt

```diff
@@ -0,0 +1,81 @@
+#
+# Copyright 2007-2015, Kaazing Corporation. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+property acceptLocation "http://localhost:8001/amqp"
+accept ${acceptLocation}
+accepted
+connected
+
+read method "GET"
+read version "HTTP/1.1"
+read header "Host" "localhost:8001"
+read header "Upgrade" /(?i:websocket)/
+read header "Connection" /(?i:Upgrade)/
+read header "Sec-WebSocket-Key" /(?<key>[a-zA-Z0-9+\/=]{24})/
+read header "Sec-WebSocket-Version" "13"
+read header "Authorization" "Basic am9lOndlbGNvbWU="
```

Comment:
Please set and use properties for this header so it's clear what the value is, as done in https://github.com/k3po/k3po/blob/develop/specification/ws/src/main/scripts/org/kaazing/specification/ws/limits/should.fail.max.lifetime.exceeded/handshake.request.and.frame.rpt. To summarise:

1. define property: `property validCredentials ${ws:base64Encode("joe:welcome")}`
1. Use it to for the header: `read header "Authorization" "Basic ${validCredentials}"
(not sure if this will work, otherwise define and use an authHeader property has done in the ws script)